js/script.js

```diff
@@ -0,0 +1,222 @@
+// All the DOM selectors stored as short variables 
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper') 
+const form = document.getElementById('name-form') 
+
+/*audio sound*/
+let sound = document.getElementById("myAudio");
+// This function will add a chat bubble in the correct place based on who the sender is
+const showMessage = (message, sender) => {
+  if (sender === 'user') {
+    chat.innerHTML += `
+      <section class="user-msg">
+        <div class="bubble user-bubble">
+          <p>${getTime()} ${message}</p>
+        </div>
+      </section>
+    `
+  } else if (sender === 'bot') {
+    chat.innerHTML += `
+      <section class="bot-msg">
+      <div class="bubble bot-bubble">
+      <p>${getTime()} ${message}</p>
+      </div>
+      </section>
+    `
+  }
+  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  chat.scrollTop = chat.scrollHeight
+}
+// Starts here
+const greeting = () => {
+  showMessage(`Welcome to our online bootcamp, <br>What is your name?`, 'bot')
+}
+//display time
+const getTime = ()=> {
+  let today = new Date();
+  let hours =  today.getHours();
+  let minutes = today.getMinutes();
+  
+  if(hours < 10){
+    hours = "0" + hours;
+  }
+  if(minutes < 10){
+    minutes = "0" + minutes;
+  }
+  let time = hours + ":" + minutes
+   return time;
+}
+//display error message
+const errorDisplay = (errorMessage)=>{
+  inputWrapper.innerHTML =`
+  <label>${errorMessage}</label> 
+  `
+}
+
+
+//email validation
+const emailValidation = (emailMessage,courseTerm)=>{
+  let emailFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if(emailMessage.match(emailFormat)){
+    setTimeout(()=>showMessage(`Thank you! You will recieve a full info-package at ${emailMessage}`, 'bot'),1000)
+    inputWrapper.innerHTML =`
+    <input type="button" value="Thanks">
+    `
+    setTimeout(()=>location.reload(), 6000)
+  }else{
+    errorDisplay(`${emailMessage} is wrong email format`)
```

Comment:
Here it would have been nice to send the user back to the input form to be able to write their email again. 

We remember you talking about this in Fridays demo. If we had not heard that - it might have been a good idea to add some comments about this function not working the way you want it to - and how you would have wanted it to work.

Reply:
Till now its not working. I will do it in future. Actually I want to avoid reload() instead of current page again.